src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -1445,12 +1445,14 @@
           ns-alias (second (re-find #"\(*(\b[a-zA-Z-.<>*=&?]+)/[a-zA-Z-]*$" line))
           line-match-suffix (first (re-find #":?([a-zA-Z-.<>*=&?]*|^\(/)$" line))
           line-prefix (subs line 0 (- (count line) (count line-match-suffix)))
-          completions (reduce (fn [ret item]
-                                (doto ret
-                                  (.push (str line-prefix item))))
+          completions (if (empty? line-match-suffix)
```

Comment:
I am surprised that the `completion-candidates` fn does not depend on the prefix item. I would expect it is it's responsibility to do the `if`. Let me try to push a commit here.

Reply:
Oh I see `completion-candidates` comes up with all the completions all the time